src/index.js

```diff
@@ -33,7 +36,9 @@ app.get('/api-docs', swaggerUi.setup(swaggerDocument));
 if (!isProduction) {
   app.use(errorhandler());
 }
-
+app.use('/api', coprofRoute);
+app.use('/api', disprofRoute);
+app.use('/api', updateprofRoute);
```

Comment:
Why can't we use `/routes/index.js` to import all routes and import a single file containing all here in index.js